js/hw9.js

```diff
@@ -0,0 +1,9 @@
+const scheckStringLength = function (a, b) {
+    if (a.length > b) {
+        return 'String is too long!';
+    }
+    else {
+        return 'Hi';
```

Comment:
read the task attentively. Your function must return accepted word if the length of it less than second argument value. In your case your function always returns `Hi`